byzcoin/bcadmin/test.sh

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-DBG_TEST=1
+DBG_TEST=0
```

Comment:
I think Linus usually asks me to check these in at level 1, so that you can see what it is testing. Not sure about this, though.

Reply:
yes, `DBG_TEST=1` rulez!